packages/react-component-library/src/tokens/TypographyTable.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import { selectors, TypographySize } from '@royalnavy/design-tokens'
+import styled from 'styled-components'
+
+const { fontSize, color, spacing } = selectors
+
+const tokens = [
```

Comment:
Not for now but I will have a look to see how we can avoid this being duplicated and hard-coded.

We should be able to change the tokens in a single place and have Storybook examples reflect that.

Reply:
#3758